api/src/main/java/ca/bc/gov/educ/eas/api/controller/v1/PENRecordController.java

```diff
@@ -0,0 +1,24 @@
+package ca.bc.gov.educ.eas.api.controller.v1;
+
+import ca.bc.gov.educ.eas.api.endpoint.v1.PENRecordEndpoint;
+import ca.bc.gov.educ.eas.api.service.v1.PENRecordService;
+import ca.bc.gov.educ.eas.api.struct.v1.StudentMerge;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+public class PENRecordController implements PENRecordEndpoint {
```

Comment:
Please rename this class & service to remove references to PEN. We're retrieving student merges.